data/csp/azure/settings/suma-server/sle15/sp5/packages.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  _namespace_azure_regionsrv_client_addon:
+    package:
+      - cloud-regionsrv-client-addon-azure
```

Comment:
We do not need this, there is no switch option from PAYG to BYOS for SUMa in Azure.

Reply:
Ok, dropped and also removed the config snippet that enables the timer.